pkg/networkmanager/interfaces.js

```diff
@@ -3716,18 +3720,18 @@ PageNetworkBondSettings.prototype = {
                     self.settings.connection.interface_name = val;
                 });
         body.find('#network-bond-settings-members')
-                .append(slaves_element = render_slave_interface_choices(model, master)
+                .replaceWith(slaves_element = render_slave_interface_choices(model, master)
```

Comment:
I don't understand this change. Can you please explain?

Reply:
Of course. Firstly, Garret has the simplified html that we should use for this modal here, view-source:https://garrett.github.io/cockpit-mockweb/forms/grid
There is the source html of the latest dialog.

The unsorted list with the settings members should not be contained in any wrapper html element as we see in garrets code. Otherwise his css will **not** work. Because of jquery I need to have however some element with id in the html, so that I will find it in the js file and replace it with the real content.

But we really need to use replaceWith so that we replace the wrapper div with the unsorted list, and not just append it to the div, so that Garrets css will work, since it expects the html elements to not be nested.